README.rst

```diff
@@ -95,6 +95,43 @@ collectd_plugin_processes_collectfiledescriptor: True
 collectd_plugin_processes_collectcontextswitch: True
 collectd_plugin_processes_collectmemorymaps: True
 
+
+collectd_plugins_write_http_*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+These vars are ones passed to the ``write_http`` plugin
+See the collectd `config guide <https://collectd.org/documentation/manpages/collectd.conf.5.shtml#plugin_write_http>`_ for details.
+
+::
+
+  collectd_plugin_write_http_nodes: {}
+  # OR
+  collectd_plugin_write_http_nodes:
+    example:
+      url: "http://example.com/collectd-post"
+      password: "pass"
+      verifypeer: true|false
```

Comment:
This might be the pattern so don't change on my behalf, but I'd expect examples to be "working" versions, and making use of default values in the example, I feel like using `true|false` is more a documentation thing, and not ideal for examples.

Reply:
This file is for documentation. There are some examples elsewhere, but it is not completely consistent